README.md

```diff
@@ -21,6 +21,14 @@ for machine learning. It features an imperative, *define-by-run* style user API.
 *define-by-run* API, the code written with Optuna enjoys high modularity, and the user of
 Optuna can dynamically construct the search spaces for the hyperparameters.
 
+## News
+
+Help us create the next version of Optuna!
+Please take a few minutes to fill in this survey, and let us know how you use Optuna now and what improvements you'd like.🤔
+:
```

Comment:
Is this colon intentional?

Reply:
What colon? 😅

Reply:
Seems fixed, Maybe I was seeing an illusion. 🙂